db/migrate/20231217000021_create_default_pet_tasks.rb

```diff
@@ -0,0 +1,11 @@
+class CreateDefaultPetTasks < ActiveRecord::Migration[7.0]
+  def change
+    create_table :default_pet_tasks do |t|
+      t.bigint :organization_id, null: false
```

Comment:
Missing index. Either create a reference or add a index

t.references :organization

or

add_index :default_pet_tasks, :organization_id

Reply:
Yes good call. Index is going to beneficial here.